### DISCUSSION TITLE
<title>archwm</title?

### DISCUSSION THREAD

**Discussion by 0x7n8** [community/community#64284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

anyone can give me i3wm or dwm config?

**Comment by 0x7n6** [comment]
https://github.com/0xZipp0/My-i3wm

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
